authors/apps/authentication/backends.py

```diff
@@ -1,10 +1,31 @@
-# import jwt
+import jwt
 #
```

Comment:
@tesh254 remove commented out lines of code

Reply:
I have removed the commented lines